doc/release-notes.md

```diff
@@ -29,51 +29,49 @@ version 0.13 should not require any additional actions.
 Downgrade warning
 -----------------
 
-### Downgrade to a version < 0.13.0.0
+### Downgrade to a version < 0.14.0.0
 
-Downgrading to a version smaller than 0.13 is not supported anymore as DIP2/DIP3 has
+Downgrading to a version smaller than 0.14 is not supported anymore as DIP8 has
 activated on mainnet and testnet.
 
-### Downgrade to versions 0.13.0.0 - 0.13.3.0
+### Downgrade to versions 0.14.0.0 - 0.14.0.1
 
-Downgrading to 0.13 releases is fully supported until DIP0008 activation but is not
-recommended unless you have some serious issues with version 0.14.
+Downgrading to older 0.14 releases is fully but is not
```

Comment:
Downgrading to older 0.14 releases is fully supported but is not

Reply:
Thanks, fixed